corehq/apps/case_importer/templates/case_importer/partials/excel_field_rows.html

```diff
@@ -3,7 +3,7 @@
 <tr>
 
 <td>
-    <!--ko if: $index() != 0-->
+    <!-- ko if: $parent.mappingRows().length > 1 -->
```

Comment:
Just to make sure i understand this correctly,
we are iterating over `mappingRows`,
earlier we were avoiding adding the remove option for the first row by checking for index,
but now we are just checking for the number of rows instead. How does that work?

Reply:
Now if there is only 1 row we can't remove it. Otherwise any row is removeable

Reply:
i think the reason the first row was ignored was what you are trying to address. The first row can not be removed because at least one should remain?

Reply:
>at least one should remain

Which means that I didn't miss something crucial. Does this solution seem reasonable to you?

Reply:
ya.. so the change you made now allows user to leave any single row instead of just the first one which sounds like an reasonable improvement.